tests/bluechi_test/machine.py

```diff
@@ -122,6 +122,22 @@ def copy_container_script(self, script_file_name: str):
              with content:\n{content}")
         self.create_file(target_dir, script_file_name, content)
 
+    def copy_machine_lib(self):
+        LOGGER.info("Copying machine lib...")
+        curr_dir = os.getcwd()
+        source_dir = os.path.join(curr_dir, "..", "..", "..", "bluechi_test", "bluechilib")
+        if not os.path.isdir(source_dir):
+            # This message is relevant for `finish` phase, where code coverage report is being created
+            LOGGER.info("bluechilib directory not found, proceeding")
+            return
+        self.exec_run("mkdir -p /tmp/bluechi_test/bluechilib")
```

Comment:
For a container setup it is necessary to copy the files each time. For the multihost setup we reuse hosts and, since we don't (and shouldn't) cleanup these files - we could do a check if the directory already exists and skip if it does.

Reply:
Done

Reply:
Done